Src/Core/CaseStudy.Core/Contracts/IReposritories/IUserFavRepo.cs

```diff
@@ -0,0 +1,17 @@
+﻿using CaseStudy.Core.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CaseStudy.Core.Contracts.IReposritories
+{
+    public interface IUserFavRepo
+    {
+        public Task<IEnumerable<Cars>> GetFavCars(int userId);
```

Comment:
no need of access modifiers for methods or properties in interface, as these can be overriden by base class.

Reply:
okay